app/uk/gov/hmrc/fileupload/controllers/transfer/TransferController.scala

```diff
@@ -67,4 +66,5 @@ class TransferController(withBasicAuth: BasicAuth,
       case Xor.Left(_) => ExceptionHandler(LOCKED, s"Envelope with id: $envelopeId locked")
     }.recover { case e => ExceptionHandler(SERVICE_UNAVAILABLE, e.getMessage) }
   }
+
 }
```

Comment:
Downloading produces the following warning:
[warn] p.c.s.n.NettyModelConversion - Content-Type set both in header (application/zip) and attached to entity (application/octet-stream), ignoring content type from entity. To remove this warning, use Result.as(...) to set the content type, rather than setting the header manually.